### DISCUSSION TITLE
可於此，提供中文詞彙或單字發音

### DISCUSSION THREAD

**Discussion by oniondelta** [oniondelta/Onion_Rime_Files#25] [Announcements]
如討論認可，於更新時添入相關字詞典。

**Comment by Haymitch-chen** [comment]
親愛的洋蔥之神，希望可以加入「䔰」ㄈㄨˊ　zᴊ⁶

**Reply by oniondelta** [reply]
原先已有「䔰」，注音為「ㄈㄨˋ」，20251001 已新添加注音「ㄈㄨˊ」。

該字符不在預設 [字符範圍](https://github.com/oniondelta/Onion_Rime_Files/wiki/%E3%80%94%E6%B4%8B%E8%94%A5%E6%B3%A8%E9%9F%B3%E7%B3%BB%E5%88%97%E3%80%95%EF%BC%9A%E7%B0%A1%E9%AB%94%E8%88%87%E5%AD%97%E7%AC%A6%E7%AF%84%E5%9C%8D%E5%92%8C%E7%B7%A8%E7%A2%BC#%E5%AD%97%E7%AC%A6%E7%AF%84%E5%9C%8D) 內，須打開〔完整〕字符範圍，才能在選單中找到！（純注音無限定字符範圍，其他版本則有）
快捷鍵「control」+「 ' 」可打開〔完整〕字符範圍。
